tor/__init__.py

```diff
@@ -1,49 +1 @@
-import bugsnag
-from addict import Dict
-
 __version__ = '2.6.14'
```

Comment:
Added the CHANGELOG.md file and CONTRIBUTING.md in a subsequent commit, per the recommendations in http://keepachangelog.com. 👍